ray_ci/bootstrap_linux.sh

```diff
@@ -5,6 +5,11 @@ export DOCKER_BUILDKIT=1
 
 cd "$RAY_REPO_DIR" || true
 
+if [ "$BUILDKITE_MESSAGE" =~ "NO_CI" ]; then
+  echo "Bypassing linux build and testing!"
+  exit 0
+fi
```

Comment:
Can we adjust the string to match the `[build_base]` condition and move it down to the other message trigger?

```suggestion
if [ "$BUILDKITE_MESSAGE" =~ "[no_ci]" ]; then
  echo "Bypassing linux build and testing!"
  exit 0
fi
```